modules/migration-installing-migration-operator-olm.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id="installing-migration-operator-with-olm_{context}"]
+= Installing the CAM Operator with OLM
+
+You can install the CAM Operator on the {product-title} 4 cluster with OLM. The CAM Operator installs the Cluster Application Migration tool on this cluster by default.
+
+.Procedure
+
+. Click *Administration* -> *Namespaces*.
+. On the *Namespaces* page:
+.. Click *Create Namespace*.
+.. Enter `openshift-migration` in the *Name* field and click *Create*.
+
+. Click *Operators* -> *OperatorHub*.
+. On the *OperatorHub* page:
+.. Scroll or type a keyword into the *Filter by keyword* field (in this case, *Migration*) to find the *Cluster Application Migration Operator*.
+.. Select the *Cluster Application Migration Operator* and click *Install*.
+
+. On the *Create Operator Subscription* page:
+.. Select the *openshift-migration* namespace if it is not already selected.
+.. Select an *Automatic* or *Manual* approval strategy.
+.. Click *Subscribe*.
+
+. Click *Catalog* -> *Installed Operators*. The *Cluster Application Migration Operator* is listed in the *openshift-migration* project with the status *InstallSucceeded*.
+
+. On the *Installed Operators* page:
+
+.. Under *Provided APIs*, click *View 12 more...*.
```

Comment:
I wouldn't call out "12 more". This might change.

Reply:
Unfortunately, we have to do this right now. There's no other way to get to the *Create New* button. Blame the OperatorHub designers....